tools/harness-simulation/harness/Thread_Harness/THCI/OpenThread_Sim.py

```diff
@@ -71,26 +78,37 @@ def __connect(self, device, node_id):
             else:
                 raise Exception('Password error')
 
-        self.__stdin, self.__stdout, _ = self.__handle.exec_command(device + ' ' + str(node_id), get_pty=True)
-        self.__stdout.channel.setblocking(0)
+        # Avoid SSH connection lost after inactivity for a while
+        self.__handle.get_transport().set_keepalive(self.KEEPALIVE_INTERVAL)
```

Comment:
Is `set_keepalive` also helpful for OpenThread_BR.py? Can we also adopt set_keepalive enhancement for OpenThread_BR (maybe in another PR) ?

Reply:
Actually, a stable network connection (packet loss rate drops from 3% to 0) contributes the major increase on SSH connection stability. `set_keepalive` may potentially make the connection more stable after a long time inactivity. Whether each component is useful or not may be experimented by ablation study. From my perspective, I think it will not be worse at least if keepalive is set.

Reply:
We had disconnection issue on real devices when the device is idle for a long time. 
This should be a straightforward fix for that.

Reply:
I will make a PR to fix the potential disconnection issue for OpenThread_BR later.